src/univariate/continuous/laplace.jl

```diff
@@ -114,8 +114,27 @@ rand(rng::AbstractRNG, d::Laplace) =
 
 #### Fitting
 
-function fit_mle(::Type{<:Laplace}, x::Array)
-    xc = copy(x)
-    a = median!(xc)
-    Laplace(a, StatsBase.mad!(xc, center=a))
+function fit_mle(::Type{<:Laplace}, x::AbstractArray{T}) where {T <: Real}
```

Comment:
The type parameter can be omitted:
```suggestion
function fit_mle(::Type{<:Laplace}, x::AbstractArray{<:Real})
```

Reply:
Note I am using the type parameter for "zero(T)" now.